app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    func_cache = {}
+
+    def wrapper(*args: Any) -> Any:
```

Comment:
```suggestion
    def wrapper(*args: tuple) -> Any:
```
Annotation is not needed in this case, or if you want do this: use **`tupple`** for args.